Document/0x06e-Testing-Cryptography.md

```diff
@@ -48,12 +95,51 @@ The following is an example of the APIs usage:
 int result = SecRandomCopyBytes(kSecRandomDefault, 16, randomBytes);
 ```
 
+Note: if other mechanims are used for random numbers in the code: verify that these are either wrappers around the APIs mentioned above review them for their secure-randomness. Often this is too hard, which means you can best stick with the implementation above.
```

Comment:
APIs mentioned above review them for their secure-randomness ->  APIs mentioned above or review them for their secure-randomness (educated guess; not 100% sure what else could be meant here?)